script.js

```diff
@@ -284,6 +284,13 @@ const EMAIL_REQUIRED = 'Please enter your email';
 const EMAIL_INVALID = 'Please enter a correct email address format';
 
 form.addEventListener('submit', (event) => {
+  const fullname = document.getElementById('name').value;
+  const email = document.getElementById('email').value;
+  const message = document.getElementById('message').value;
+
+  const formData = { fullname, email, message };
+
+  window.localStorage.setItem('formData', JSON.stringify(formData));
   // stop form submission
   event.preventDefault();
```

Comment:
Currently the form is not saved when you enter the data and reload. It only gets saved when after entering the data and clicking on the button to submit. However, as stated in the project requirements, the data should be saved when the user changes the content of any input field (and not when it is submitted).
Kindly check and update it accordingly


![Screenshot from 2023-04-14 21-37-37](https://user-images.githubusercontent.com/65233860/232159233-6fc20917-aa3b-426a-ba97-97a1e9869c96.png)